controllers/admin/AdminModulesController.php

```diff
@@ -74,198 +54,23 @@ public function __construct()
-        /* 		$this->list_modules_categories['search_filter']['name'] = $this->trans('Search & Filter', [], 'Admin.Modules.Feature'); */
-        $this->list_modules_categories['seo']['name'] = $this->trans('SEO', [], 'Admin.Catalog.Feature');
-        $this->list_modules_categories['shipping_logistics']['name'] = $this->trans('Shipping & Logistics', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['slideshows']['name'] = $this->trans('Slideshows', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['smart_shopping']['name'] = $this->trans('Comparison site & Feed management', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['market_place']['name'] = $this->trans('Marketplace', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['others']['name'] = $this->trans('Other Modules', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['mobile']['name'] = $this->trans('Mobile', [], 'Admin.Global');
-        $this->list_modules_categories['dashboard']['name'] = $this->trans('Dashboard', [], 'Admin.Global');
-        $this->list_modules_categories['i18n_localization']['name'] = $this->trans('Internationalization & Localization', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['emailing']['name'] = $this->trans('Emailing & SMS', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['social_networks']['name'] = $this->trans('Social Networks', [], 'Admin.Modules.Feature');
-        $this->list_modules_categories['social_community']['name'] = $this->trans('Social & Community', [], 'Admin.Modules.Feature');
-
-        uasort($this->list_modules_categories, [$this, 'checkCategoriesNames']);
-
         // Set Id Employee, Iso Default Country and Filter Configuration
         $this->id_employee = (int) $this->context->employee->id;
-        $this->iso_default_country = $this->context->country->iso_code;
-        $this->filter_configuration = Configuration::getMultiple([
-            'PS_SHOW_TYPE_MODULES_' . (int) $this->id_employee,
-            'PS_SHOW_COUNTRY_MODULES_' . (int) $this->id_employee,
-            'PS_SHOW_INSTALLED_MODULES_' . (int) $this->id_employee,
-            'PS_SHOW_ENABLED_MODULES_' . (int) $this->id_employee,
-            'PS_SHOW_CAT_MODULES_' . (int) $this->id_employee,
-        ]);
-    }
-
-    public function checkCategoriesNames($a, $b)
-    {
-        if ($a['name'] === $this->trans('Other Modules')) {
-            return true;
-        }
-
-        return (bool) ($a['name'] > $b['name']);
     }
 
     public function setMedia($isNewTheme = false)
     {
         parent::setMedia($isNewTheme);
         $this->addJqueryPlugin(['autocomplete', 'fancybox', 'tablefilter']);
-
-        if ($this->context->mode == Context::MODE_HOST && Tools::isSubmit('addnewmodule')) {
-            $this->addJS(_PS_JS_DIR_ . 'admin/addons.js');
-        }
-    }
-
-    public function displayAjaxRefreshModuleList()
```

Comment:
Please document these BC (and all below)

Reply:
Done